lib/_emerge/actions.py

```diff
@@ -1000,11 +1000,18 @@ def unresolved_deps():
 				msg.append("    %s" % (parent,))
 				msg.append("")
 			msg.extend(textwrap.wrap(
-				"Have you forgotten to do a complete update prior " + \
-				"to depclean? The most comprehensive command for this " + \
+				"Have you forgotten to resolve unsatisfied dependencies prior "
+				"to depclean? The simplest possible command for this "
 				"purpose is as follows:", 65
 			))
 			msg.append("")
+			msg.append("  " + \
+				good("emerge @unsatisfied-deps"))
```

Comment:
It turns out that `@unsatisfied-deps` is often unsuitable here because it
pulls in a bunch of `@installed` packages, when you really want to use
`@world` as the source of truth.

Reply:
My plan is it introduce an `@unsatisfied-world` set which is equivalent to `@unsatisfied-deps` but filters out any non @world packages.